package.json

```diff
@@ -78,7 +697,7 @@
         "controlType": "textbox"
       },
       {
-        "key": "RETURN_DATE",
+        "key": "returnDate",
```

Comment:
The format of this field should be validated in the index.js and trigger an error if the format is not valid.
Hint use RegExp